packages/orchestration/README.md

```diff
@@ -68,6 +83,25 @@ const response = await orchestrationClient.chatCompletion({
 const responseContent = response.getContent();
 ```
 
+`getContent()` is a convenience method that parses the response and returns the model's output as a string.
+
+#### Customizing the Request Configuration
```

Comment:
Please consider removing this section as it is the task from the other PR #188.

https://github.com/SAP/ai-sdk-js/pull/188#discussion_r1784074411